### DISCUSSION TITLE
Scaling between species in MoBi

### DISCUSSION THREAD

**Discussion by RobinM92** [Open-Systems-Pharmacology/Forum#1154] [Questions & Problems]
Dear community,

A more general question; what is the generally accepted workflow to scale across species in MoBi? E.g. do you export the different species from PKSim to Mobi, save and load their PSV, MSV and Spatial structure or do you need to adjust more parameters? 

Thanks again!
Kind regards,
Robin

**Comment by RobinM92** [comment]
I see that this is quite a topic of discussion within the 'core team', has any consensus already been reached here?

https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/378
https://github.com/Open-Systems-Pharmacology/R-Toolbox/issues/39 
https://github.com/Open-Systems-Pharmacology/OSMOSES/issues/16

**Comment by RobinM92** [comment]
Dear community, any insights on this topic? 
Thanks!
Robin

**Reply by StephanSchaller** [reply]
That's still the state of things and the way to go is to use the human model as a basis and then make sure you overwrite the parameters correctly for other species (a workflow we take care of in R).

**Comment by RobinM92** [comment]
Thanks a lot Stephan! I assume transferring simulations for the 
different species from PKSim to separate MoBi instances and implementing 
the new processes would also work, but perhaps using R to just overwrite 
a human model would also be a good idea (faster or less error-prone), is 
this described somewhere?

**Reply by StephanSchaller** [reply]
Hi Robin, no, there is no description. Just compare the list of parameters created when creating the different individuals in R, then it will be quite obvious and do a verification by comparing it to simulation results from an actual animal simulation created in PK.Sim.
